docs/Middlewares.md

```diff
@@ -8,7 +8,7 @@ Fastify provides out of the box an asynchronous [middleware engine](https://gith
 
 Fastify middlewares don't support the full syntax `middleware(err, req, res, next)`, because error handling is done inside Fastify.
 
-Also, if you are using Middlewares that bundles different small middlewares, such as [*helmet*](https://helmetjs.github.io/), we recommend to use the single modules to get better performances.
+Also, if you are using a middleware that bundles different, smaller middlewares, such as [*helmet*](https://helmetjs.github.io/), we recommend to use the single modules to get better performances.
```

Comment:
I don't think the comma is needed here.

Reply:
In between "also" and "if"? Or "different" and "smaller"? I think if it's the latter, commas are always necessary in between adjectives.

Reply:
Between "different" and "smaller". /shrug I'm not an English professor. I just doesn't seem right to me.

Reply:
If we were English professors then how could we help build dope NodeJS web services? 😅